source/installation/config.html.md

```diff
@@ -44,18 +44,13 @@ MAIL_USER=example
 MAIL_PASSWORD=123456
 MAIL_HELO_NAME=example.org
 MAIL_ADDRESS=from@example.org
+```
+
+For **Windows** systems, everything is the same, unless you are using SQLite for your database, then you will need to change your paths
+from backslashes (\) to forward slashes (/) else you will receive error.
```

Comment:
This remark is useless, since both use `/`.

Reply:
Excellent thank you.

Reply:
Removed.